xblockutils/public/studio_container.js

```diff
@@ -0,0 +1,55 @@
+function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
+    var $buttons = $(".add-xblock-component-button", element);
+
+    function isSingleInstance(button) {
+        return $(button).data('single-instance');
+    }
+
+    $buttons.click(function(ev) {
```

Comment:
You're reconstructing the jquery object several times in this function. Could you just do `$(this)` once and reuse it?